packages/@aws-cdk/core/test/test.app.ts

```diff
@@ -125,8 +125,11 @@ export = {
 },
 
 'setContext(k,v) can be used to set context programmatically'(test: Test) {
-  const prog = new App();
-  prog.node.setContext('foo', 'bar');
+  const prog = new App({
+    context: {
+      foo: 'bar'
+    }
+  });
   test.deepEqual(prog.node.tryGetContext('foo'), 'bar');
```

Comment:
I have this in `test.synthesis.ts`. Put it here since this is a test of what gets synthesized on different options, even though it's a property passed into `App`. What do you think?